### PR TITLE
Added a landing page for the AI/ML webinar

### PR DESCRIPTION
**Pull request by caldav**
## Done

* Created a landing page for the AI/ML webinar on Aug 22
* [Copydoc](https://docs.google.com/document/d/1YVPlQYT3jvxb6Yp5OShSiUsmKBPOYOm_fNLJHNsQ6tQ/edit#) 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001//engage/ai-ml-ubuntu](http://0.0.0.0:8001//engage/ai-ml-ubuntu)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure the page looks nice and registration works

## Notes

* Design review appreciated

### CODE REVIEW

File: templates/engage/ai-ml-ubuntu.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+{% block title %}AI, ML and Ubuntu: Everything you need to know{% endblock %}
+
+{% block content %}
+<section class="p-strip p-takeover--ai-webinar">
+  <div class="row u-equal-height">
+    <div class="col-9">
+      <h1 class="p-takeover__title">Artificial intelligence, machine learning and Ubuntu</h1>
+      <p class="p-takeover__text">Webinar live on Aug 22 -
+        <a href="#register-section">Register now</a>
```
Comment (anthonydillon) [inline templates/engage/ai-ml-ubuntu.html]: This should either be a button or have a `&nbsp;&rsaquo;` on it.

**Comment by caldav** [comment]
@anthonydillon indeed. Since it's more of a pointer than a CTA (we don't want to chainlink CTAs), I went with rsaquo, thanks!

File: templates/engage/ai-ml-ubuntu.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{% extends "engage/base_engage.html" %}
+
+{% block meta_description %}Ready to make the migration from proprietary virtualisation to OpenStack?{% endblock %}
+
+{% block title %}AI, ML and Ubuntu: Everything you need to know{% endblock %}
+
```
Comment (Caleb-Ellis) [inline templates/engage/ai-ml-ubuntu.html]: Can you add a copydoc metadata link here?
`{% block meta_copydoc %}https://docs.google.com/document/d/1YVPlQYT3jvxb6Yp5OShSiUsmKBPOYOm_fNLJHNsQ6tQ/edit{% endblock meta_copydoc %}`

File: templates/engage/ai-ml-ubuntu.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      </ul>
+      <h3>Other questions answered</h3>
+      <ul>
+        <li>What do Kubeflow, Tensorflow, Jupyter, and GPGPUs do?</li>
+        <li>What’s the difference between AI, ML and DL?</li>
+        <li>What is an AI model? How do you train it? How do you develop / improve it? How do you execute it?</li>
+      </ul>
+    </div>
+    <div class="col-6">
+      <img src="https://assets.ubuntu.com/v1/1793c307-slide-preview.png" alt="AI, ML and Ubuntu webinar - slides preview">
```
Comment (Caleb-Ellis) [inline templates/engage/ai-ml-ubuntu.html]: Replace `https://assets.ubuntu.com/v1/` with `{{ ASSET_SERVER_URL }}` variable

File: templates/engage/ai-ml-ubuntu.html (hunk 2 above)
Comment (caldav) [reply templates/engage/ai-ml-ubuntu.html]: Done!

**Comment by kwm14** [comment]
@caldav landing page looks good 👍 I was going to recommend an update on the hero but didn't realise it linked from the AI takeover on the homepage 😄

**Pull request merged**